app/src/main/java/com/asfoundation/wallet/topup/LocalTopUpPaymentPresenter.kt

```diff
@@ -96,7 +97,7 @@ class LocalTopUpPaymentPresenter(
 
   private fun onViewCreatedRequestLink() {
     disposables.add(localPaymentInteractor.getTopUpPaymentLink(packageName, data.fiatValue,
-        data.fiatCurrencyCode, paymentId)
+        data.fiatCurrencyCode, paymentId, context?.getString(R.string.topup_title) ?: "Top up")
```

Comment:
This should not be hard coded

Reply:
only if context is null, which should not happen